### DISCUSSION TITLE
potential new data source for global electricity stats

### DISCUSSION THREAD

**Discussion by Cammille-Mitchell** [electricitymaps/electricitymaps-contrib#6244] [Ideas] [answered]
I came across the IEA (International Energy Association) which is an organization that partners with 47 countries to share energy data. They publish a monthly report on electricity statistics among the Organization for Economic Co-operation and Development (OECD) member countries. Would this data source be beneficial ? 
Link: https://www.iea.org/data-and-statistics/data-product/monthly-electricity-statistics#schedule

**Comment by VIKTORVAV99** [comment, marked as answer]
Hi and thanks for sharing the source!

We are already using this internally along with a few other sources to make sure the hourly production values we get from our parsers are aligned with IEA in this case.

But since this source don't have hourly (or better) data available we can't use it directly.
